reference/halyard/secrets/s3-secrets.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: single
+title:  "Secrets in S3"
+sidebar:
+  nav: reference
+---
+
+{% include toc %}
+
+
+This document describes how to set up Spinnaker secrets in an encrypted S3 bucket. In this example, we'll be using a `mybucket` bucket in the `us-west-2` region to store GitHub credentials and a kubeconfig file. We'll be referencing the bucket by its URL `mybucket.us-west-2.amazonaws.com`.
+
+
+## Authorization
+Since we're storing sensitive information, we'll protect the bucket by restricting access and [enabling encryption](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/default-bucket-encryption.html).
```

Comment:
```suggestion
Since you're storing sensitive information, you protect the bucket by restricting access and [enabling encryption](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/default-bucket-encryption.html).
```